packages/react-vapor/src/components/editor/CodeEditor.tsx

```diff
@@ -62,11 +73,13 @@ export class CodeEditor extends React.Component<ICodeEditorProps, CodeEditorStat
     }
 
     componentDidUpdate(prevProps: ICodeEditorProps) {
+        this.props.isCollapsibleExpanded && this.editor.refresh();

```

Comment:
Won't that trigger some useless refresh when the value changes? I think it would be better to do it this way:

```suggestion
        if (!prevProps.isCollapsibleExpanded && this.props.isCollapsibleExpanded) {
            this.editor?.refresh()
        }
```

Reply:
It wouldn't work, because the componentDidUpdate is called twice when we open the collapsible, and the second time is the moment the editor has to be refreshed. However, I found an alternative solution, in which I check if the codeEditor was refresh less then twice.